gef.py

```diff
@@ -6116,7 +6214,7 @@ def fastbin_index(sz):
         for i in range(NFASTBINS):
             gef_print("Fastbins[idx={:d}, size={:#x}] ".format(i, (i+1)*SIZE_SZ*2), end="")
             chunk = arena.fastbin(i)
-            chunks = []
+            chunks = set()
```

Comment:
I don't think a `set()` will work. Depending on the case, you can poison the fastbins to have the same chunk multiple times.

Reply:
Yes, but that's how we're getting it to work: We are keying it by the address, and the second there's a collision we print `[loop detected]` and then we break out.

Reply:
Let's go with it. I'll leave myself a note for the future, there is something that bothers me here, can't really figure out.